### DISCUSSION TITLE
Made a translation into Ukrainian

### DISCUSSION THREAD

**Discussion by otkachov** [Ylianst/MeshCentral#6335] [Show and tell]
Hi all,
I translated MeshCentral to my language Ukrainian.
File translate.json sent to e-mail specified in "MeshCentral Translator".

Could you add the Ukrainian language in the next release?

**Comment by si458** [comment]
Hi sure! I'll give @Ylianst a nudge!
If u also send it to myself (check my gitub profile page for email) I can also add it in too :)

**Comment by otkachov** [comment]
It is done :)
I sent this email to you with file _translate.json_

Thank you!

**Comment by si458** [comment]
All done :)
